genfragments/ThirteenPointSixTeV/Higgs/HH/HHto2G4Q_TuneCP5_13p6TeV_madgraph-pythia8.py

```diff
@@ -0,0 +1,38 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunesRun3ECM13p6TeV.PythiaCP5Settings_cfi import *
+from Configuration.Generator.PSweightsPythia.PythiaPSweightsSettings_cfi import *
+
+generator = cms.EDFilter("Pythia8ConcurrentHadronizerFilter",
+                         maxEventsToPrint = cms.untracked.int32(1),
+                         pythiaPylistVerbosity = cms.untracked.int32(1),
+                         filterEfficiency = cms.untracked.double(1.0),
+                         pythiaHepMCVerbosity = cms.untracked.bool(False),
+                         comEnergy = cms.double(13600.),
+                         PythiaParameters = cms.PSet(
+        pythia8CommonSettingsBlock,
+        pythia8CP5SettingsBlock,
+        pythia8PSweightsSettingsBlock,
+        processParameters = cms.vstring(
+            '24:mMin = 0.05', # the lower limit of the allowed mass range generated by the Breit-Wigner (in GeV)
+            '24:onMode = off', # Turn off all W decays 
+            '24:onIfAny = 1 2 3 4 5', # Add W->qq decays 
+            '25:onMode = off', # Turn off all H decays 
+            '25:onIfMatch = 22 22', # Add H->gg decay
+            '25:onIfMatch = 24 -24', # Add H->WW decay 
```

Comment:
you don't want H->ZZ->4Q here

Reply:
That follows the run2 request. To @wangjiannju @hqucms answer please

Reply:
Thank you very much for the comment! I added genfragments/ThirteenPointSixTeV/Higgs/HH/HHto2G2Zto4Q_TuneCP5_13p6TeV_madgraph-pythia8.py for H->ZZ->4Q @menglu21  @acarvalh

Reply:
then maybe change the name to HHto2G2Wto4Q